plasma_framework/test/endToEndTests/QuasarExit.e2e.test.js

```diff
@@ -1485,6 +1544,776 @@ contract(
+
+                                    describe('If both Bob and Carol now wish to withdraw funds from the pool', () => {
+                                        before(async () => {
+                                            this.bobBalanceBeforeWithdraw = new BN(await web3.eth.getBalance(bob));
+                                            this.carolBalanceBeforeWithdraw = new BN(await web3.eth.getBalance(carol));
+                                            this.bobQTokenBalance = await this.qEth.balanceOf(bob);
+                                            this.carolQTokenBalance = await this.qEth.balanceOf(carol);
+                                            this.bobWithdrawTx = await this.quasar.withdrawFunds(
+                                                ETH,
+                                                this.bobQTokenBalance,
+                                                { from: bob },
+                                            );
+
+                                            this.carolWithdrawTx = await this.quasar.withdrawFunds(
+                                                ETH,
+                                                this.carolQTokenBalance,
+                                                { from: carol },
+                                            );
+                                        });
+
+                                        it('should be able to withdraw all qTokens', async () => {
+                                            const bobQTokenBalanceAfterWithdraw = await this.qEth.balanceOf(bob);
+                                            const carolQTokenBalanceAfterWithdraw = await this.qEth.balanceOf(carol);
+
+                                            expect(bobQTokenBalanceAfterWithdraw).to.be.bignumber.equal(new BN(0));
+                                            expect(carolQTokenBalanceAfterWithdraw).to.be.bignumber.equal(new BN(0));
+                                        });
+
+                                        it('should update the quasar capacity and balance', async () => {
+                                            const quasarCapacityAfterWithdraw = new BN(
+                                                await this.quasar.tokenUsableCapacity(ETH),
+                                            );
+                                            const quasarBalance = new BN(
+                                                await web3.eth.getBalance(this.quasar.address),
+                                            );
+                                            const { poolSupply } = await this.quasar.tokenData(ETH);
+
+                                            expect(quasarCapacityAfterWithdraw).to.be.bignumber.equal(
+                                                quasarBalance,
+                                            ).equal(poolSupply);
+                                            expect(quasarBalance).to.be.bignumber.at.most(
```

Comment:
Why are we expecting the balance to be >= 0 and <= 10? Is it not deterministic?

Reply:
This is the small change that might remain in the pool because of the rounding off. again the amount is supposed to be low and 0 - 10 is a guess. Is there a better way we could do this?

Reply:
Ah ok, that's fine. Maybe add a comment explaining to our future selves 😄